airbyte-integrations/connectors/source-leadfeeder/integration_tests/acceptance.py

```diff
@@ -0,0 +1,16 @@
+#
+# Copyright (c) 2024 Airbyte, Inc., all rights reserved.
```

Comment:
If this is just boilerplate, I would advise to delete the whole integration_tests directory and adjust acceptance-test-config.